### PR TITLE
Allow setting `remainPresentFor` in Publisher's ConnectionConfiguration

### PR DESCRIPTION
**Pull request by JakubJankowski**
Resolves [455](https://github.com/ably/ably-asset-tracking-swift/issues/455) by allowing users to set the `remainPresentFor` property on a ConnectionConfiguration object.

Should allow users to fix an issue when a rider enters to or exits from a channel intermittently and is still aware of rider presence continually.

### CODE REVIEW

File: Sources/AblyAssetTrackingInternal/Configuration/ConnectionConfiguration+Extensions.swift [hunk 1]
```diff
@@ -27,6 +27,11 @@ extension ConnectionConfiguration {
             clientOptions.environment = environment
         }
         
+        if let remainPresentForMilliseconds = remainPresentForMilliseconds {
```
Comment (lawrence-forooghian) [inline Sources/AblyAssetTrackingInternal/Configuration/ConnectionConfiguration+Extensions.swift:30]: Could we add a test to `Tests/InternalTests/Configuration/ConnectionConfiguration+ExtensionsTests.swift` which tests that it correctly sets the `transportParams` when `remainPresentInMilliseconds` is specified?

**Review by lawrence-forooghian — COMMENTED** [review]
Looks good, just one comment.

File: Sources/AblyAssetTrackingInternal/Configuration/ConnectionConfiguration+Extensions.swift (hunk 1 above)
Comment (JakubJankowski) [reply Sources/AblyAssetTrackingInternal/Configuration/ConnectionConfiguration+Extensions.swift:30]: Good point, added in 06526ce6fd20524221d48851b6b26da65f80026c

Comment (JakubJankowski) [reply Sources/AblyAssetTrackingInternal/Configuration/ConnectionConfiguration+Extensions.swift:30]: Good point, added in 06526ce6fd20524221d48851b6b26da65f80026c

**Review by lawrence-forooghian — APPROVED** [review]

**Pull request merged**